storybook/src/components/Tabs/Tabs.docs.mdx

```diff
@@ -18,3 +18,9 @@ The first tab is active by default.
 Another tab’s panel can be displayed initially as well.
 
 <Canvas of={TabsStories.WithInitialTab} />
+
+### With wide content
+
+The Tab Buttons list and the content within the active Tab Panel will scroll horizontally when their width exceeds the available space.
```

Comment:
This may not be accurate because the table is doing the scrolling

Reply:
No, the Tab Panel first scrolls its content if it would otherwise overflow. If that content is a Table, it no longer needs to scroll because its width is no longer constrained.

Reply:
Did you try? I'll send you a video on slack.